### PR TITLE
Test cases translation scripts

### PR DESCRIPTION
**Pull request by ashutoshsingh0223**
Scripts for converting english test cases(yaml) to csv for translation  and the back to yaml after translation.

### CODE REVIEW

File: test_case_translation/generate_translated_tests.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+import io
```
Comment (srajdev) [inline test_case_translation/generate_translated_tests.py]: W1618 [34mimport missing `from __future__ import absolute_import`[0m

File: test_case_translation/yaml_to_csv.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+import yaml
```
Comment (srajdev) [inline test_case_translation/yaml_to_csv.py]: W1618 [34mimport missing `from __future__ import absolute_import`[0m

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS FAILED... Please resolve issues

**Comment by haptik-deployment** [comment]
UNIT TESTS FAILED... Please resolve issues

**Comment by haptik-deployment** [comment]
UNIT TESTS FAILED... Please resolve issues

File: test_case_translation/generate_translated_tests.py [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+def my_represent_none(self, data):
+    """
+    yaml representer for NoneType
+    """
+    return self.represent_scalar(u"tag:yaml.org,2002:null", u"null")
+
+
+
+def main():
```
Comment (srajdev) [inline test_case_translation/generate_translated_tests.py]: E303 too many blank lines (3)

File: test_case_translation/generate_translated_tests.py [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+    file_name = yaml_filepath.split(".yaml")[0]
+    yaml_data = yaml.load(io.open(yaml_filepath, "r", encoding="utf-8"), Loader=yaml.Loader)
+    yaml_tests = []
+    args = yaml_data.get("args", None)
+    for language in yaml_data["tests"]:
+        yaml_tests.extend(yaml_data["tests"][language])
+
+    csv_filepath = sys.argv[2]
+    csv_data = pd.read_csv(csv_filepath, encoding="utf-8")
+    csv_rows = csv_data.to_dict(orient="records")
```
Comment (srajdev) [inline test_case_translation/generate_translated_tests.py]: F841 local variable 'csv_rows' is assigned to but never used

File: test_case_translation/generate_translated_tests.py [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+    csv_data["id"] = csv_data["unique_id"].apply(lambda x: x.split("-")[0])
+    csv_data["output_id"] = csv_data["unique_id"].apply(lambda x: int(x.split("-")[1]))
+
+
+    translated_data = []
```
Comment (srajdev) [inline test_case_translation/generate_translated_tests.py]: E303 too many blank lines (2)

File: test_case_translation/generate_translated_tests.py [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+    else:
+        if yaml_tests:
+            raise Exception(
+                "Empty translated data. {filename} might not be in proper format".format(filename=csv_filepath))
+        elif translated_data:
+            raise Exception("Empty yaml tests {filename} might not be in proper format".format(filename=yaml_filepath))
+        else:
+            raise Exception("{filename1} and {filename2} might not be proper format".format(filename1=yaml_filepath,
+                                                                                            filename2=csv_filepath))
+
```
Comment (srajdev) [inline test_case_translation/generate_translated_tests.py]: W293 blank line contains whitespace

File: test_case_translation/generate_translated_tests.py [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+        if yaml_tests:
+            raise Exception(
+                "Empty translated data. {filename} might not be in proper format".format(filename=csv_filepath))
+        elif translated_data:
+            raise Exception("Empty yaml tests {filename} might not be in proper format".format(filename=yaml_filepath))
+        else:
+            raise Exception("{filename1} and {filename2} might not be proper format".format(filename1=yaml_filepath,
+                                                                                            filename2=csv_filepath))
+   
+main()
```
Comment (srajdev) [inline test_case_translation/generate_translated_tests.py]: E305 expected 2 blank lines after class or function definition, found 1

File: test_case_translation/generate_translated_tests.py [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+                "Empty translated data. {filename} might not be in proper format".format(filename=csv_filepath))
+        elif translated_data:
+            raise Exception("Empty yaml tests {filename} might not be in proper format".format(filename=yaml_filepath))
+        else:
+            raise Exception("{filename1} and {filename2} might not be proper format".format(filename1=yaml_filepath,
+                                                                                            filename2=csv_filepath))
+
+
+main()
+
```
Comment (srajdev) [inline test_case_translation/generate_translated_tests.py]: W391 blank line at end of file

**Comment by haptik-deployment** [comment]
UNIT TESTS FAILED... Please resolve issues

**Pull request closed without merging**